src/context_diff.rs

```diff
@@ -422,13 +427,15 @@ mod tests {
                                 if f != 2 {
                                     bet.write_all(b"l\n").unwrap();
                                 }
+                                let _ = File::create(&format!("{target}/aalef")).unwrap();
+                                let mut fa = File::create(&format!("{target}/alef")).unwrap();
```

Comment:
Do you really need to create dummy files here (and in other tests)?
Isn't that a leftover from when the `get_modification_time()` function didn't handle well non-existent file paths?

Reply:
Yupp, Right!